src/main/java/edu/harvard/iq/dataverse/search/SearchIncludeFragment.java

```diff
@@ -205,6 +205,7 @@ public String searchRedirect(String dataverseRedirectPage, Dataverse dataverseIn
          */
         
         dataverse = dataverseIn;
+        logger.info("redirect page: "+dataverseRedirectPage);
```

Comment:
Should lower to fine.

Reply:
Not sure why it was set to .info in my brach...

Reply:
Gone now, great. Thanks.